data/schema/core/actionwml.cfg

```diff
@@ -480,7 +480,7 @@
 		{DEFAULT_KEY slowed s_bool no}
 		{DEFAULT_KEY petrified s_bool no}
 		{DEFAULT_KEY unhealable s_bool no}
-		{DEFAULT_KEY experience s_bool no}
+		{DEFAULT_KEY experience string_list yes}
 		{DEFAULT_KEY resistance_modifier s_real 1}
```

Comment:
this should be resistance_multiplier

Reply:
Yes - I'm okay with fixing it in this PR, or with fixing it separately.